lib/linguist/languages.yml

```diff
@@ -2356,6 +2356,19 @@ Lex:
   tm_scope: none
   ace_mode: text
   language_id: 199
+Lightning Component:
+  type: markup
+  ace_mode: html
+  codemirror_mode: htmlmixed
+  codemirror_mime_type: text/html
+  extensions:
+    - ".auradoc"
+    - ".cmp"
+    - ".design"
+    - ".evt"
+  language_id: 432
+  tm_scope: none
```

Comment:
You've entered `html` for the `ace_mode`. Couldn't HTML also be used for the `tm_scope` (grammar for source highlighting)?